gems/smithy/lib/smithy/plan.rb

```diff
@@ -6,35 +6,62 @@ class Plan
     # @param [Hash] model The API model as a JSON hash.
     # @param [Symbol] type The type of code to generate, either :client, :server, or :schema.
     # @param [Hash] options The options passed to the generator.
+    # rubocop:disable Metrics/AbcSize
     def initialize(model, type, options = {})
```

Comment:
Should we document the options for initialize?

Reply:
Yeah, I think we should. They are also on the command line, but I see cases where we want them here too.